core/src/main/java/org/apache/calcite/avatica/remote/Driver.java

```diff
@@ -59,7 +63,8 @@ public Driver() {
    */
   public enum Serialization {
     JSON,
-    PROTOBUF;
+    PROTOBUF,
+    LOOKER;
```

Comment:
I agree with the comment below, that serialization doesn't feel quite right here. You're replacing the endpoint, not just the wire format. I'm not sure what alternative to suggest though as it probably requires bigger refactoring to encapsulate the logic of pluggable back ends. Whether it's worth it or not depends on whether this code is expected to get upstreamed, and I don't know the intentions here.

Reply:
https://github.com/looker-open-source/calcite-avatica/pull/2/commits/761d6d64f1cc011906cfd4b0d0c426ecb145778e removes this `serialization` bit and adds a new Looker-specific `org.apache.calcite.avatica.remote.looker.Driver` implementation. This can be used via `jdbc:looker` as the URL protocol without changing the behavior of `jdbc:avatica:remote` connections.